acceptance/indy_acceptance/indy-tests/test_results/Acceptance_Test_20171117_17-42-58/Test_Scenario_09_Remove_And_Add_Role_20171117_17-42-58.json

```diff
@@ -0,0 +1,173 @@
+{
+  "testcase": "Test_Scenario_09_Remove_And_Add_Role",
```

Comment:
It looks like a log. I think it needs to be removed

Reply:
We removed it.